index.js

```diff
@@ -87,7 +87,7 @@ function _sanitize(target, options) {
   };
 }
 
-function sanitize(target, options) {
+function sanitize(target = {}, options = {}) {
```

Comment:
I don't think the target should be optional - it doesn't make sense to call sanitize without any parameters!

```suggestion
function sanitize(target, options = {}) {
```